### DISCUSSION TITLE
Delay in development for Inquiry Specific Environment

### DISCUSSION THREAD

**Discussion by Nadiya496** [Sunbird-inQuiry/Community#34] [General Discussions]
Hello @chitranshu-keshav,@sharathkashyap, @luckynvdu

Apologies for the delay in the development of inquiry specific BB. Currently in our BB front end editors, player and backend services are deployed but unfortunately publish flink jobs we are facing soem issues and currently team is working on the CSP interrupts. Henceforth backend services development we are targetting to complete in the release 5.2.0. Therefore we will not be able to do inquiry specific regression testing. For now new features of 5.1 will be validated in the creation portal before main regression.

cc: @pallakartheekreddy @surendrasinghs @anunagaraj @Aparna-karri591 @ShailandraOjha @krgauraw @rajnishdargan @itsvick

**Comment by Aparna-karri591** [comment]
Hello @chitranshu-keshav,
Apologies for the delay in sing off, due to deployment failure issue in inquiry portal and one defect we are not giving the sign off on 26-09-2022, Tomorrow morning we will try to give the sign off.
CC: @Nadiya496 @anunagaraj @ShailandraOjha @pallakartheekreddy @surendrasinghs
